commit 5586e53285674b11192fd0eb7e39517d48165f6e (refs/changes/90/4790/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2018-09-05T11:07:31-07:00 (1 year, 1 month ago)
    
    TRITON-753 imgapi test suite is adding image test files with the wrong ownership

diff --git a/test/reload-test-data.sh b/test/reload-test-data.sh
index 7097a54..c6679c0 100755
--- a/test/reload-test-data.sh
+++ b/test/reload-test-data.sh
@@ -125,4 +125,6 @@ elif [[ "$opt_mode" == "dc" ]]; then
         echo "icon" >$icon_path
         i=$(($i + 1))
     done
+    # Change image ownership, as imgapi runs as user 'nobody'.
+    chown -R nobody:nobody /data/imgapi/images
 fi
